### ISSUE TITLE
FR (math branch): Improve the presentation of the images

### ISSUE THREAD

**Issue by ignaciogros** [exelearning/iteexe#55]
PNG images based on LaTeX or MathML code should have a better presentation.

No additional options are required, just a simple and clean presentation with smooth dark text on a white (or transparent?) background. Maybe some padding too. See how https://www.mathjax.org/ displays mathematical markup.

See handleTinyMCEmath and handleTinyMCEmathML in mainpage.py.
